gulpfile.js

```diff
@@ -787,13 +824,13 @@ function compileJs(srcDir, srcFilename, destDir, options) {
       .pipe(lazywrite())
       .on('end', function() {
         appendToCompiledFile(srcFilename, destDir + '/' + destFilename);
-        logBuildStep('Compiled', srcFilename);
-      }));
+      })).then(() => {
+        endBuildStep('Compiled', srcFilename, startTime);
```

Comment:
@erwinmombay / @cramforce / @choumx / @jridgewell: It turns out the time being measured here is the duration between when the task is initially queued and when it eventually completes. Ideally, I'd like to start the time measurement when the task execution begins, and not when it's queued.

Is there a "task started" event I can latch on to, where the timer can be started?